app/controllers/subjects_controller.rb

```diff
@@ -5,6 +5,24 @@ def show
     @reductions = workflow.subject_reductions.where(subject_id: subject.id)
   end
 
+  def search
+    skip_authorization
+    @workflow = workflow
+    search_params = params.require(:search).permit(:id)
+    subject_id = search_params[:id]
+
+    if Subject.exists?(subject_id)
+      @subject = Subject.find(subject_id) || Subject.new
+      @extracts = workflow.extracts.where(subject_id: subject_id)
+      @reductions = workflow.subject_reductions.where(subject_id: subject_id)
+
+      render 'show'
+    else
+      @subject = Subject.new(id: subject_id)
```

Comment:
Does this not have the same security implications as https://github.com/zooniverse/caesar/pull/314 ?

Reply:
It's a good thing to be on the alert for, but all I use this subject for is to display the message "subject with [id] not found." It's never persisted to the database.

Reply:
Seems like an awkward and heavy way to just pass an integer to the view, but I'm not as familiar with the view bits as you are so I'll defer.

Reply:
i thought about just making an OpenStruct that responds to `:id` but it seemed hacky.